transport_plugins/bled112/iotile_transport_bled112/async_packet.py

```diff
@@ -24,7 +26,14 @@ def __init__(self, filelike, header_length, length_function):
         self.file = filelike
         self._stop = Event()
 
-        self._thread = Thread(target=ReaderThread, args=(filelike, self.queue, header_length, length_function, self._stop))
+        dedupe = False
```

Comment:
To keep the interface to this class clean, I would recommend moving these assignments out into the main `BLED112DeviceAdapter` class and just passing in kwargs of `dedupe` and `dedupe_timeout` with defaults:

```
def __init__(..., deduplicate=False, deduplicate_timeout=0)